File: go/prime.go
package main

import (
	"fmt"
	"math"
)

func main() {
	max := int(1e6)

	for num := 3; num < max; num += 2 {
		prime := true
		sqrt := int(math.Sqrt(float64(num)))

		for i := 3; i <= sqrt; i += 2 {
			if num%i == 0 {
				prime = false
				break
			}
		}

		if prime {
			fmt.Println(num)
		}

	}

}

File: java/prime.java

class Prime {
    public static void main(String[] args) {
        double max = 1000000;

        for (int num = 3; num < max; num += 2)
        {
            int prime = 1;
            double sqr = Math.sqrt(num);
            
            for(int i=3;i<=sqr;i +=2){
                if(num %i ==0){
                    prime = 0;
                    break;
                }
            }
            if(prime==1){
                System.out.println(num);
            }               
        }    
    }
}